contrib/mod_tls.c

```diff
@@ -10674,13 +10677,28 @@ MODRET tls_prot(cmd_rec *cmd) {
     return PR_ERROR(cmd);
   }
 
+  c = find_config(main_server->conf, CONF_PARAM, "TLSOptions", FALSE);
+  while (c != NULL) {
+    unsigned long opts = 0;
+
+    pr_signals_handle();
+    opts = *((unsigned long *) c->argv[0]);
+    tls_opts |= opts;
+
+    c = find_config_next(c, c->next, CONF_PARAM, "TLSOptions", FALSE);
+  }
```

Comment:
Frankly, I've been looking at the source how to get that value, and I've copy-pasted this part from another function.

So, I should just remove the tls_opts local variable and simply use the global one?